docs/modules/operation/pages/database-reports/templates/maintenance-stragegy.adoc

```diff
@@ -1,20 +1,50 @@
 [[maintenance-strategy-report]]
 = Maintenance Contracts Strategy Report
 
+== Report Contents
+This assessment management report is designed to assist in creating and reviewing a network management maintenance strategy.  This report has a pie chart showing the number of assets in the various contract state (e.g., active contract, expired contract, no contract, etc.)
```

Comment:
Use "for example" instead of "e.g."
```suggestion
This assessment management report is designed to assist in creating and reviewing a network management maintenance strategy.
This report has a pie chart showing the number of assets in the various contract state (for example, active contract, expired contract, no contract, etc.)
```